WeatherToday/Configurations/Configuration.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Configuration.swift
+//  WeatherToday
+//
+//  Created by Taher on 10/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Foundation
+
+enum Configuration: String {
+
+    case debugStaging
+    case debugProduction
+    case releaseStaging
+    case releaseProduction
+
+    // MARK: - Current Configuration
+
+    static let current: Configuration = {
+        #if DEBUG && PRODUCTION
+        return .debugProduction
+        #elseif DEBUG && STAGING
+        return .debugStaging
+        #elseif RELEASE && STAGING
+        return .releaseStaging
+        #elseif RELEASE && PRODUCTION
+        return .releaseProduction
+        #endif
+    }()
+
+    static var baseWeatherURL: String {
+        guard let urlValue = Bundle.main.infoDictionary?["WEATHER_API_BASE_URL"] as? String else {
+            fatalError("No URL Found")
```

Comment:
I think we should return empty string and print out comment rather than crashing it. Because this value will be call when user do some API action so the app will not fail at launch which is harder to debug when we are developing (we have to call API to check if app crash or not)

Reply:
updated in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/20/commits/0df0b8e6ef1945dbb46c9bd130e52d369b861265)